src/Page/Partners/Partner_.elm

```diff
@@ -199,13 +203,17 @@ partnerLogo : Maybe String -> String -> Html msg
 partnerLogo maybeLogoUrl partnerName =
     case maybeLogoUrl of
         Just logoUrl ->
-            div [ css [ partnerLogoContainer ] ]
-                [ img [ src logoUrl, css [ partnerLogoStyle ], alt (partnerName ++ " logo") ] []
-                , hr [ css [ hrStyle ] ] []
-                ]
+            if isValidUrl logoUrl then
+                div [ css [ partnerLogoContainer ] ]
+                    [ img [ src logoUrl, css [ partnerLogoStyle ], alt (partnerName ++ " logo") ] []
+                    , hr [ css [ hrStyle ] ] []
+                    ]
+
+            else
+                text ""
 
         Nothing ->
-            div [] []
+            text ""
```

Comment:
Just needs a valid HTML node returned, of which TEXT is basically "" where as the `<div></div>` seemed a bit clunky

Reply:
And nothing in the styles relied on it being there? It all still lays out as expected?

Reply:
Not really. It's just all elm methods must return a value of the correct type (otherwise i'd return a null). Its a pattern that katja uses in a few places